src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx

```diff
@@ -1,9 +1,684 @@
   return (
     <>
-      <Text variant="xl">Auction Partnerships App</Text>
+      <MetaTags
+        title="Auction Partnerships"
+        description=""
+        pathname="/auction-partnerships"
+      />
+
+      <Join
+        separator={<Spacer my={6} border="1px solid" borderColor="black15" />}
+      >
+        <ArtsyForAuctions />
+        <MarketplaceExperience />
+        <PartnerWithArtsyForm />
+        <GlobalAudienceReach />
+        <BiddingTools />
+        <ConsignmentsInfo />
+        <VisibilityInfoTop />
+        <VisibilityInfoBottom />
+        <AuctionsSupportTeam />
+      </Join>
     </>
   )
 }
+
+const ArtsyForAuctions: React.FC = () => {
+  const headerImage =
+    "http://files.artsy.net/images/auctionpartnerships-lowres-header-temp.png"
+
+  return (
+    <Box>
+      <FullBleedHeader src={headerImage} />
+      <Spacer mb={4} />
+      <GridColumns gridRowGap={4}>
+        <Column span={6}>
+          <Text variant="xl">Artsy for Auctions</Text>
+        </Column>
+        <Column span={6}>
+          <Text variant="lg">
+            Artsy offers a centralized online platform to promote your auction,
+            expand your client base, maximize bids, and support auction
+            operations with our suite of cloud-based and mobile tools.
+          </Text>
+        </Column>
+      </GridColumns>
+    </Box>
+  )
+}
+
+const MarketplaceExperience: React.FC = () => {
+  const image = resized(
+    "http://files.artsy.net/images/gallery-names/temp.jpeg",
+    {
+      width: 910,
+      height: 345,
+    }
+  )
+
+  return (
+    <GridColumns>
+      <Column span={6}>
+        <ResponsiveBox aspectWidth={4300} aspectHeight={1632} maxWidth="100%">
+          <Image
+            src={image.src}
+            width="100%"
+            height="100%"
+            srcSet={image.srcSet}
+            lazyLoad
+            alt=""
+          />
+        </ResponsiveBox>
+      </Column>
+      <Column span={6}>
+        <Text variant="sm" textTransform="uppercase" mb={1}>
+          Experience
+        </Text>
+        <Text variant="lg" mb={1}>
+          Trusted Marketplace for Both Commercial and Benefit Auctions
+        </Text>
+        <Text variant="sm">
+          Our roster of global auction house partners is rapidly expanding,
+          including, Bonhams, Phillips, Heritage, Rago/Wright, and Hindman
+          Auctions. To date, Artsy benefit auctions have helped to raise more
+          than $25 million for museums, nonprofit organizations, and other
+          charitable causes.
+        </Text>
+      </Column>
+    </GridColumns>
+  )
+}
+
+const PartnerWithArtsyForm: React.FC = () => {
+  useLoadScript({
+    id: "marketo-form",
+    src: "https://app-ab14.marketo.com/js/forms2/js/forms2.min.js",
+    onReady: () => {
+      // @ts-ignore
+      window.MktoForms2.loadForm(
+        "//app-ab14.marketo.com",
+        "609-FDY-207",
+        SHOULD_WARN_PRODUCTION_ENVIRONMENT
```

Comment:
can we just use `getENV("NODE_ENV") === 'production'`

Reply:
Staging will have a NODE_ENV of production though.

I was thinking we should rename this variable though.

Reply:
ohhh i see i see 👍 np.